server/repository/assessment/updateCountryStatus.ts

```diff
@@ -1,25 +1,31 @@
 import { BaseProtocol, DB, Schemas } from '@server/db'
 import { Objects } from '@core/utils'
 import { CountryIso } from '@meta/area'
-import { AssessmentName, CountryStatus, Cycle } from '@meta/assessment'
+import { AssessmentName, Cycle } from '@meta/assessment'
+import { CountryStatus } from '@meta/area/country'
 
 export const updateCountryStatus = async (
-  props: { countryStatus: CountryStatus; countryIso: CountryIso | string; name: AssessmentName; cycleName: string },
+  props: {
+    countryStatus: CountryStatus
+    countryIso: CountryIso | string
+    assessmentName: AssessmentName
+    cycleName: string
+  },
   client: BaseProtocol = DB
 ): Promise<CountryStatus | undefined> => {
-  const { countryStatus, countryIso, name, cycleName } = props
+  const { countryStatus, countryIso, assessmentName, cycleName } = props
 
-  const assessmentCycleName = Schemas.getNameCycle({ props: { name } }, { name: cycleName } as Cycle)
+  const assessmentCycleName = Schemas.getNameCycle({ props: { name: assessmentName } }, { name: cycleName } as Cycle)
 
   return client.oneOrNone<CountryStatus>(
     `
-        update ${assessmentCycleName}.country_status
-        set status = $2, desk_study = $3
+        update ${assessmentCycleName}.country
+        set props = props || $2
         where country_iso = $1
-        returning *;
+        returning country_iso, props->>'status' as status, (props->>'desk_study')::boolean as desk_study;
```

Comment:
why returning this? why not all country row?

Reply:
This is to separate the refactoring of countryStatus to another PR... otherwise this PR will become hard to keep up
So this is just a placeholder to return type of CountryStatus